docs/docs/APIGuide/Losses.md

```diff
@@ -2186,3 +2186,62 @@ loss = criterion.forward(input, target)
 > loss
 -6.1750183
 ```
+
+
+## TransformerCriterion ##
+**Scala:**
+```scala
+val criterion = TransformerCriterion(criterion, Some(inputTransformer), Some(targetTransformer))
+```
+**Python:**
+```python
+criterion = TransformerCriterion(criterion, input_transformer, targetTransformer)
+```
+
+The criterion that takes two modules (optional) to transform input and target, and take
```

Comment:
Elaborate the purpose of this criterion in doc. e.g. "This criterion is used for ..." or in what situations.

Reply:
@shane-huang Added